src/NHSD.BuyingCatalogue.Ordering.Api/Extensions/OrderExtensions.cs

```diff
@@ -56,5 +56,30 @@ public static bool IsAssociatedServicesSectionComplete(this Order order)
         {
             return !(order is null) && order.AssociatedServicesViewed;
         }
+
+        public static bool IsSectionStatusComplete(this Order order, int serviceRecipientsCount, int catalogueSolutionsCount, int associatedServicesCount)
+        {
+            var scenario1 = catalogueSolutionsCount > 0 
```

Comment:
I wonder if we can come up with better naming for the variables here. Scenario doesn't give the reader much context as to what the result is meant to entail. Also it looks like `IsFundingSourceComplete` is common across all scenarios. So that could be checked first?